libass/ass_parse.c

```diff
@@ -610,14 +610,18 @@ char *ass_parse_tags(RenderContext *state, char *p, char *end, double pwr,
                 v2 = argtod(args[1]);
             } else
                 continue;
-            if (state->evt_type & EVENT_POSITIONED) {
+
+            int allow_transform = state->renderer->track->PositionTransform;
+
+            if (state->evt_type & EVENT_POSITIONED && 
+                (!allow_transform || (allow_transform && !nested))) {
```

Comment:
I’d be inclined to align the behaviour to other transformable tags and just have repeated tags reset the value:

```suggestion
                !allow_transform) {
```